model/adjuster/otel_tag.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2023 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adjuster
+
+import (
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+
+	"github.com/jaegertracing/jaeger/model"
+)
+
+var otelLibraryKeys = map[string]struct{}{
+	string(semconv.OTelLibraryNameKey):    {},
+	string(semconv.OTelLibraryVersionKey): {},
+}
+
+func OTelTagAdjuster() Adjuster {
+	return Func(func(trace *model.Trace) (*model.Trace, error) {
```

Comment:
nit: please move to a helper function to reduce nesting (you have 5 levels of indentation)

Reply:
moved it to an anonymous function